### PR TITLE
Alter the test configurations to allow java8 bytecode in tests

### PR DESCRIPTION
**Pull request by AdamBJohnsonx**

### CODE REVIEW

**Review by shahzaibj — APPROVED** [review]

File: uiautomationutilities/build.gradle [hunk 1]
```diff
@@ -47,7 +60,6 @@ android {
 
 
     defaultConfig {
-        multiDexEnabled true
```
Comment (iambmelt) [inline uiautomationutilities/build.gradle:50]: No longer needed?

Comment (AdamBJohnsonx) [reply uiautomationutilities/build.gradle:50]: doesn't seem to be?  If it is, isn't it mainly important for desugaring?

Comment (iambmelt) [reply uiautomationutilities/build.gradle:50]: multiDex (as I understand it) is usually a workaround for method counts exceeding the apk dex file limit

Comment (shahzaibj) [reply uiautomationutilities/build.gradle:50]: Might be needed for UI Automator I think

File: testutils/build.gradle [hunk 2]
```diff
@@ -185,7 +197,12 @@ dependencies {
     implementation "com.nimbusds:nimbus-jose-jwt:$rootProject.ext.nimbusVersion"
     implementation 'androidx.appcompat:appcompat:1.0.2'
     implementation "junit:junit:$rootProject.ext.junitVersion"
-    implementation ("org.robolectric:robolectric:$rootProject.ext.robolectricVersion")
+    implementation ("org.robolectric:robolectric:$rootProject.ext.robolectricVersion") {
+        exclude group: "org.apache.maven"
+        exclude group: "org.apache.maven.wagon"
+        exclude group: "xmlpull"
+        exclude group: "xpp3"
```
Comment (gitaumoses4) [inline testutils/build.gradle:204]: Why do we need to exclude these? 😀

**Review by gitaumoses4 — APPROVED** [review]
Looks good.

File: testutils/build.gradle (hunk 2 above)
Comment (AdamBJohnsonx) [reply testutils/build.gradle:204]: These are causing our duplicate classes.

File: uiautomationutilities/build.gradle [hunk 3]
```diff
@@ -47,7 +60,6 @@ android {
 
 
     defaultConfig {
```
Comment (AdamBJohnsonx) [inline uiautomationutilities/build.gradle:62]:
```suggestion
    defaultConfig {
        multiDexEnabled true
```

File: uiautomationutilities/build.gradle (hunk 1 above)
Comment (AdamBJohnsonx) [reply uiautomationutilities/build.gradle]: I just put it back.  It shouldn't hurt anything anyway.

File: testutils/build.gradle (hunk 2 above)
Comment (gitaumoses4) [reply testutils/build.gradle:204]: Oh. I see.

**Pull request merged**